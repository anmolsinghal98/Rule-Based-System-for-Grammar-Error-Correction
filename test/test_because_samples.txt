he is good because. 
he is good because work. 
he works because he is good. 
he is late because me. 
he is late because of nothing.
Because he went to the mall.
I think that I should put comic books in the library. I have two reasons. First, if the library has comic books, many people will go to the library. Second, I can enjoy myself at the library. Because I like to feel and see different books. That is why I think I should put comic books in the library.
I think magazines should be put in libraries. It is. Because magazines are able to know a lot of things, as many as books. And so on. Magazines are not only fun, but we can see pictures. There are pictures in magazines. I think they moved people. Because they are drawn pictures. So, I think we should read magazines if everyone can follow the manners.
We are living in unprecedented times. The Prime Minister needs to come forward and address the nation because of this. Unless he assures everyone that the country is united against the fight with the invisible enemy, we will not be able to flourish because it. 