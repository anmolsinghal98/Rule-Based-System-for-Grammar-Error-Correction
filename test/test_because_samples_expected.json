[
  {
    "Sample No": 1,
    "Detections": [
      {
        "Sentence ID": 0,
        "Word Index": "4",
        "Word String": "because",
        "Error Type": "Because Without Principal Clause"
      }
    ]
  },
  {
    "Sample No": 2,
    "Detections": [
      {
        "Sentence ID": 0,
        "Word Index": "4",
        "Word String": "because",
        "Error Type": "Because Without Principal Clause"
      }
    ]
  },
  {
    "Sample No": 3,
    "Detections": []
  },
  {
    "Sample No": 4,
    "Detections": [
      {
        "Sentence ID": 0,
        "Word Index": "4",
        "Word String": "because",
        "Error Type": "Because Without Principal Clause"
      }
    ]
  },
  {
    "Sample No": 5,
    "Detections": []
  },
  {
    "Sample No": 6,
    "Detections": [
      {
        "Sentence ID": 0,
        "Word Index": "1",
        "Word String": "Because",
        "Error Type": "Because Without Principal Clause"
      }
    ]
  },
  {
    "Sample No": 7,
    "Detections": [
      {
        "Sentence ID": 4,
        "Word Index": "1",
        "Word String": "Because",
        "Error Type": "Because Without Principal Clause"
      }
    ]
  },
  {
    "Sample No": 8,
    "Detections": [
      {
        "Sentence ID": 2,
        "Word Index": "1",
        "Word String": "Because",
        "Error Type": "Because Without Principal Clause"
      },
      {
        "Sentence ID": 7,
        "Word Index": "1",
        "Word String": "Because",
        "Error Type": "Because Without Principal Clause"
      }
    ]
  },
  {
    "Sample No": 9,
    "Detections": [
      {
        "Sentence ID": 2,
        "Word Index": "25",
        "Word String": "because",
        "Error Type": "Because Without Principal Clause"
      }
    ]
  }
]